packages/itmat-commons/src/models/study.ts

```diff
@@ -27,7 +29,7 @@ export interface IRole {
     studyId: string;
     name: string;
     permissions: string[];
-    users: string[];
+    users: Models.UserModels.IUser[];
```

Comment:
I think this would not be correct. Why did you change this? This does not correspond to what is stored in the mongodb. and you didn't map it back

Reply:
@fguitton 
ok. I intended this IRole to be like in mongodb entry. This IUser[] field has an extra graphql resolver on Role type that resolves the strings into IUser. The type is defined in itmat-interface/graphql/schema.js. 

Are you using this interface in the UI / any code that requires users key to be IUser[]?  If needed, could you make something like

"""
interface IRoleFromGraphql extends Omit<IRole, 'users'> {
    users: IUser[]
}
"""